sw/device/silicon_creator/rom/bootstrap.c

```diff
@@ -333,16 +333,16 @@ static rom_error_t bootstrap_handle_program(bootstrap_state_t *state) {
 }
 
 hardened_bool_t bootstrap_requested(void) {
-  uint32_t res =
-      otp_read32(OTP_CTRL_PARAM_OWNER_SW_CFG_ROM_BOOTSTRAP_EN_OFFSET);
-  if (launder32(res) != kHardenedBoolTrue) {
+  uint32_t bootstrap_dis =
+      otp_read32(OTP_CTRL_PARAM_OWNER_SW_CFG_ROM_BOOTSTRAP_DIS_OFFSET);
+  if (launder32(bootstrap_dis) == kHardenedBoolTrue) {
```

Comment:
Another hardening question here. It used to be `!=` and now it's `==`.

It looks like it was easier to trigger the early return in the prior version with a bitflip, when the condition was `if (launder32(res) != kHardenedBoolTrue) { ... }`.

Reply:
That's the goal of this PR -- we would like to be able to bootstrap even if we cannot program that word for some reason.